### ISSUE TITLE
Build a layer-2 system

### ISSUE THREAD

**Issue by vanphandinh** [graphprotocol/mission-control-curator#181]
I think we need to build a layer-2 system for Everest.link to lower the interaction fees that are too high on layer-1 Ethereum.
I suggest building on top of Matic or xDai because of those EVM-compatible sidechains so we couldn't have to change our smart-contracts

**Comment by yanivtal** [comment]
Yes we should definitely do this. I have Matic and Arbitrum as current front runners. Looking for someone who can help experiment with this.

**Comment by vanphandinh** [comment]
> Yes we should definitely do this. I have Matic and Arbitrum as current front runners. Looking for someone who can help experiment with this.

I can be a volunteer :D
